docs/aws-region-enable.md

```diff
@@ -27,24 +27,33 @@ This document provides the information on how to address and resolve missing AWS
     In the dialog box, review the informational text and choose Enable Region.
     Wait until the Region is ready to use.
 
-    We would also like to get AWS public access enabled for this region. Please use the following command:
+    We would like to get AWS public access enabled for this region. Please use the following command:
         "aws ec2 disable-image-block-public-access --region `<missing-region>`"
 
-    Let's also adjust the quota to match the other regions :
-        "aws service-quotas request-service-quota-increase --region=`<missing-region>` --service-code=ec2 --quota-code=L-0E3CBAB9 --desired-value 1000.00"
+    We would also like the quota level to be set, the quota codes can be viewed using the below command:
+        "aws service-quotas --service-code ec2 list-service-quotas"
+
+    Let's make sure to adjust the quota to match the other regions if necessary. If the current quota is less than 1000, let's bump it to at least 1000 with the following request:
+        "aws service-quotas request-service-quota-increase --region=ap-southeast-5 --service-code=ec2 --quota-code=L-0E3CBAB9 --desired-value 1000.00"
```

Comment:
Move region to the end so it's easy to update when copy/pasting.

```suggestion
        "aws service-quotas request-service-quota-increase  --service-code=ec2 --quota-code=L-0E3CBAB9 --desired-value 1000.00 --region=ap-southeast-5"
```

Reply:
Updated the command to have the region at the end